docs/webhooks/choice_webhook.md

```
@@ -0,0 +1,202 @@
+# ChoiceBank Webhook Integration
+
+This document describes how to integrate with the `choice` webhook endpoint that validates requests using the agreed upon signature generation algorithm.
+
+## Endpoint
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)